<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProduct</class>
 <widget class="QDialog" name="AddProduct">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>目的地:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>航班号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEnd">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>出发地:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtStart">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登机口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLocation">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMonth">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>起飞时间:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtYear">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>月</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDay">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>日</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHour">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>时</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMinute">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>分</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>航班座位数:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRaw">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>排</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtColumn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>280</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>列</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>头等舱范围:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRawHeadS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>排 至</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRawHeadE">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>排</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>头等舱票价:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHeadPrice">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>经济舱票价:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEcoPrice">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid rgb(41, 57, 85);   
    border-radius: 3px;  
    background: white;  
    selection-background-color: green; 
    font-size: 16px ;  
}
 
QLineEdit:hover {
    border: 1px solid blue;  
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddProduct</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddProduct</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
